eventsource/format-field-retry-bogus-2.any.js

```diff
@@ -0,0 +1,19 @@
+// META: title=EventSource: "retry" field (bogus)
```

Comment:
Let's make it at least a little clear how it's different.
```suggestion
// META: title=EventSource: "retry" field (bogus, shorter timeout)
```